client/platform/web-girder/App.vue

```diff
@@ -26,8 +26,8 @@ export default defineComponent({
   components: {},
   setup(_, { root }) {
     async function loadMetadata(datasetId: string): Promise<DatasetMeta> {
-      const ds: VIAMEDataset = await root.$store.dispatch('Dataset/load', datasetId);
-      return ds.meta;
+      const dataset: VIAMEDataset = await root.$store.dispatch('Dataset/load', datasetId);
```

Comment:
This type coercion is a lie.  dispatch() returns an any because we aren't doing proper typescript vuex.

Not a big deal, but also why I don't like vuex.